webapp/src/components/MeetingAccordion.tsx

```diff
@@ -3,15 +3,16 @@ import {
   AccordionDetails,
   AccordionSummary,
   Link as MuiLink,
-  ListItem,
-  ListItemText,
+  Typography,
 } from '@mui/material';
+
```

Comment:
Normally there's only one empty line in the imports section between the absolute and relative imports.

Reply:
Removed.